hatchways/team-bluejay
# Moshe generate user profile page #17

Ticker #17 

Description: 

- Added profile page to display user's profile and a Map of their location
- Users can edit their profile by clicking a button to open a modal
- Edited profiles are saved to user state in AuthContext
- I also updated all npm packages to latest versions
- added `client/.env` to .gitignore

Testing: 

- npm install
- add `MOSHES_GOOGLE_MAPS_API_KEY = "AIzaSyAN3q6E8D80h38QQsiSWQ5keW6YSRMPqYc"` to client/.env
- login and use navbar to visit profile page
- click to edit user profile and verify profile page is updated

I cut corners and pushed off some tasks to do later
Bugs/Todos:

- About Me in '/profile' does not display multiple paragraphs
- No validation yet for input fields
